apps/web/src/components/ListItem/list-item.tsx

```diff
@@ -14,7 +14,7 @@ interface ListItemProps {
 }
 
 const listItemVariant = cva(
-  'text-base flex items-center justify-between border-b py-2 md:py-3 gap-2',
+  'text-base flex items-center justify-between border-b py-2 md:py-[3px] gap-2',
```

Comment:
this is creating issue in other parts of the website (for instance, in course presentation page where it is used for course infos), so you should find another way to avoid this issue